camkes.cmake

```diff
@@ -380,7 +380,7 @@ function(DeclareCAmkESRootserver adl)
         1
         CAMKES_ROOT
         "" # Option arguments
-        "DTS_FILE_PATH" # Single arguments
+        "DTS_FILE_PATH;DTB_FILE_PATH" # Single arguments
```

Comment:
This should also add a `get_absolute_list_source_or_binary` to match the behavior of DTS_FILE_PATH.

Reply:
Looking at the code, I think `get_absolute_list_source_or_binary()`is never called, because the check is broken:
```
if(${CAmkESDTS} AND NOT "${CAMKES_ROOT_DTS_FILE}" STREQUAL "")
    get_absolute_list_source_or_binary(CAMKES_ROOT_DTS_FILE_PATH "${CAMKES_ROOT_DTS_FILE_PATH}")
endif()
```
It uses `CAMKES_ROOT_DTS_FILE`, but the actual variable is `CAMKES_ROOT_DTS_FILE_PATH`.

Reply:
Fixed checks and call `get_absolute_list_source_or_binary()` for DTS/DTB.